<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Agent_Cmd_Log_Form</class>
 <widget class="QWidget" name="Agent_Cmd_Log_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agent_Cmd_Log_Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRequestTelemetry">
       <property name="toolTip">
        <string>Request Telemetry</string>
       </property>
       <property name="text">
        <string>RTele</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPacketN">
       <property name="toolTip">
        <string>PacketN</string>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbComma">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbAgentN">
       <property name="toolTip">
        <string>AgentN</string>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePushCMD">
       <property name="toolTip">
        <string>Send Custom CMD to Agent</string>
       </property>
       <property name="text">
        <string>@HW</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbBS">
        <property name="text">
         <string>BS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbTS">
        <property name="text">
         <string>TS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbTL">
        <property name="text">
         <string>TL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbOD">
        <property name="text">
         <string>OD</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>537</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="pteAgentLog"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
